data_safe_haven/infrastructure/programs/sre/mirror_manager.py

```diff
@@ -0,0 +1,434 @@
+import json
+from collections.abc import Mapping
+
+from pulumi import ComponentResource, Input, Output, ResourceOptions
+from pulumi_azure_native import containerinstance, storage
+from pulumi_random import RandomPassword
+
+from data_safe_haven.config.config_sections import ConfigSubsectionGiteaMirror
+from data_safe_haven.infrastructure.common import (
+    DockerHubCredentials,
+    get_ip_address_from_container_group,
+)
+from data_safe_haven.infrastructure.components import (
+    FileShareFile,
+    FileShareFileProps,
+    LocalDnsRecordComponent,
+    LocalDnsRecordProps,
+    PostgresqlDatabaseComponent,
+    PostgresqlDatabaseProps,
+    WrappedLogAnalyticsWorkspace,
+)
+from data_safe_haven.resources import resources_path
+from data_safe_haven.utility import FileReader
+
+
+class SREGiteMirrorManagerProps:
```

Comment:
```suggestion
class SREGiteaMirrorManagerProps:
```

Also, I suggest calling the file `gitea_mirror_manager.py`

Reply:
Thanks for catching the typo! I will rename the file too